src/lightning_app/runners/cloud.py

```diff
@@ -106,6 +106,84 @@ def _to_clean_dict(swagger_object, map_attributes):
 class CloudRuntime(Runtime):
     backend: Union[str, CloudBackend] = "cloud"
 
+    def open(self, name: str, cluster_id: Optional[str] = None):
+        """Method to open a CloudSpace with the root folder uploaded."""
+        try:
+            # Check for feature support
+            user = self.backend.client.auth_service_get_user()
+            if not user.features.code_tab:
+                logger.error(
+                    "The `lightning open` command has not been enabled for your account. "
+                    "To request access, please contact support@lightning.ai"
+                )
+                sys.exit(1)
+
+            # Dispatch in four phases: resolution, validation, spec creation, API transactions
+            # Resolution
+            cloudspace_config = self._resolve_config(name, load=False)
```

Comment:
Why don't we load it ?

Reply:
For this we don't want to create or maintain a `.lightning` file since there is no expectation of re-running over an existing cloudspace, so we avoid creating or loading one